src/bindings/swig/python/tests/test_key.py

```diff
@@ -22,35 +22,34 @@ def test_ctor(self):
 		self.assertIsInstance(k, kdb.Key)
 		self.assertTrue(k.isValid())
 
-		k = kdb.Key("spec/key")
+		k = kdb.Key("spec:/key")
 		self.assertIsInstance(k, kdb.Key)
 		self.assertTrue(k.isValid())
 
-		k = kdb.Key("proc/key")
+		k = kdb.Key("proc:/key")
 		self.assertIsInstance(k, kdb.Key)
 		self.assertTrue(k.isValid())
 
-		k = kdb.Key("dir/key")
+		k = kdb.Key("dir:/key")
 		self.assertIsInstance(k, kdb.Key)
 		self.assertTrue(k.isValid())
 
-		k = kdb.Key("user/key")
+		k = kdb.Key("user:/key")
 		self.assertIsInstance(k, kdb.Key)
 		self.assertTrue(k.isValid())
 
-		k = kdb.Key("system/key")
+		k = kdb.Key("system:/key")
 		self.assertIsInstance(k, kdb.Key)
 		self.assertTrue(k.isValid())
 
 		k = kdb.Key()
 		self.assertIsInstance(k, kdb.Key)
-		self.assertFalse(k.isValid())
+		self.assertTrue(k.isValid())
```

Comment:
Because of various changes (foremost the fact that `keyNew (NULL)` now returns `NULL`, i.e. returns an error) `isValid` should now always return true. Also IMO it was pretty weird that `k` was invalid immediately after calling the constructor.